<?xml version="1.0"?>
<launch>
  <node pkg="pcd2gridmap" type="pcd2gridmap" name="pcd2gridmap_node" output="screen">
    <param name="pcd_file" value="$(find pcd2gridmap)/data/room.pcd"/>
    <param name="resolution" value="0.05"/>
    <param name="min_z" value="0.1"/>
    <param name="max_z" value="0.15"/>
    <param name="frame_id" value="map"/>
    <param name="map_topic" value="map"/>
  </node>
</launch>